Makefile

```diff
@@ -64,7 +64,7 @@ IMG ?= $(IMAGE_TAG_BASE):v$(VERSION)
 # Produce CRDs that work back to Kubernetes 1.11 (no version conversion)
 CRD_OPTIONS ?= "crd:trivialVersions=true,preserveUnknownFields=false"
 # ENVTEST_K8S_VERSION refers to the version of kubebuilder assets to be downloaded by envtest binary.
-ENVTEST_K8S_VERSION = 1.21
+ENVTEST_K8S_VERSION = 1.28.0
```

Comment:
Updated this because of OLM CI failure

Reply:
Was it the `unable to find a version that was supported for platform darwin/arm64` error? (Since it's a nice benefit 🙂 )

Reply:
```
Run make deploy-olm
/home/runner/work/gatekeeper-operator/gatekeeper-operator/ci-tools/bin/operator-sdk olm install --version v0.18.2 --timeout 5m
time="2023-09-29T23:[4](https://github.com/stolostron/gatekeeper-operator/actions/runs/6358351157/job/17270490210#step:8:5)3:08Z" level=info msg="Fetching CRDs for version \"v0.18.2\""
time="2023-09-29T23:43:08Z" level=info msg="Fetching resources for resolved version \"v0.18.2\""
time="2023-09-29T23:43:09Z" level=info msg="Creating CRDs and resources"
time="2023-09-29T23:43:09Z" level=info msg="  Creating CustomResourceDefinition \"catalogsources.operators.coreos.com\""
time="2023-09-29T23:43:09Z" level=info msg="  Creating CustomResourceDefinition \"clusterserviceversions.operators.coreos.com\""
time="2023-09-29T23:43:09Z" level=info msg="  Creating CustomResourceDefinition \"installplans.operators.coreos.com\""
time="2023-09-29T23:43:09Z" level=info msg="  Creating CustomResourceDefinition \"operatorconditions.operators.coreos.com\""
time="2023-09-29T23:43:09Z" level=info msg="  Creating CustomResourceDefinition \"operatorgroups.operators.coreos.com\""
time="2023-09-29T23:43:09Z" level=info msg="  Creating CustomResourceDefinition \"operators.operators.coreos.com\""
time="2023-09-29T23:43:09Z" level=info msg="  Creating CustomResourceDefinition \"subscriptions.operators.coreos.com\""
time="2023-09-29T23:43:09Z" level=info msg="  Creating Namespace \"olm\""
time="2023-09-29T23:43:09Z" level=info msg="  Creating Namespace \"operators\""
time="2023-09-29T23:43:09Z" level=info msg="  Creating ServiceAccount \"olm/olm-operator-serviceaccount\""
time="2023-09-29T23:43:09Z" level=info msg="  Creating ClusterRole \"system:controller:operator-lifecycle-manager\""
time="2023-09-29T23:43:09Z" level=info msg="  Creating ClusterRoleBinding \"olm-operator-binding-olm\""
time="2023-09-29T23:43:09Z" level=info msg="  Creating Deployment \"olm/olm-operator\""
time="2023-09-29T23:43:09Z" level=info msg="  Creating Deployment \"olm/catalog-operator\""
time="2023-09-29T23:43:09Z" level=info msg="  Creating ClusterRole \"aggregate-olm-edit\""
time="2023-09-29T23:43:09Z" level=info msg="  Creating ClusterRole \"aggregate-olm-view\""
time="2023-09-29T23:43:09Z" level=info msg="  Creating OperatorGroup \"operators/global-operators\""
time="2023-09-29T23:43:09Z" level=fatal msg="Failed to install OLM version \"v0.18.2\": failed to create CRDs and resources: no matches for kind \"OperatorGroup\" in version \"operators.coreos.com/v1\""
make: *** [Makefile:26[5](https://github.com/stolostron/gatekeeper-operator/actions/runs/6358351157/job/17270490210#step:8:6): deploy-olm] Error 1
```